src/Service/VendorService/UserUpload/UserUploadVendorService.php

```diff
@@ -6,40 +6,15 @@
 
 namespace App\Service\VendorService\UserUpload;
 
-use App\Entity\Vendor;
-use App\Exception\UnknownVendorServiceException;
-use App\Service\VendorService\ProgressBarTrait;
 use App\Service\VendorService\VendorServiceInterface;
 use App\Service\VendorService\VendorServiceTrait;
-use App\Utils\Message\VendorImportResultMessage;
 
 /**
  * Class UserUploadVendorService.
  */
 class UserUploadVendorService implements VendorServiceInterface
 {
-    use ProgressBarTrait;
     use VendorServiceTrait;
 
     protected const VENDOR_ID = 15;
-
```

Comment:
Why this change? and how do it relate to an new importer?

Reply:
I see....

Reply:
The refactoring of VendorServiceInterfaces means the `load()` function can be removed. Moving `getVendor()` to `VendorServiceTrait`made sense to me bacause it contains all other `getVendor*` functions.